Fixes for managing content views chap (#1637)
theforeman/foreman-documentation: guides/common/modules/proc_creating-a-composite-content-view.adoc

To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-a-composite-content-view_{context}[].

.Procedure

> . In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create content View*.

Reviewer: ```suggestion
. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create content view*.
```

. In the *Name* field, enter a name for the view.
{ProjectName} automatically completes the *Label* field from the name you enter.
. In the *Description* field, enter a description of the view.

> . Select the *Composite View?* checkbox to create a Composite Content View.

Reviewer: According to [BZ#2123134](https://bugzilla.redhat.com/show_bug.cgi?id=2123134), there is no **Composite View?** checkbox.

Author: > According to [BZ#2123134](https://bugzilla.redhat.com/show_bug.cgi?id=2123134), there is no **Composite View?** checkbox.

I double checked. the outlined steps work in the UI

Author: > According to [BZ#2123134](https://bugzilla.redhat.com/show_bug.cgi?id=2123134), there is no **Composite View?** checkbox.

good catch

To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-a-composite-content-view_{context}[].

.Procedure
. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create content view*.

> . In the *Create content view* pop-up window, enter a name for the view in the *Name* field.

Reviewer 2: I'm not sure if we should use "pop-up window", maybe I'd go with just "window".
(I haven't found any other usage of "pop-up window" in the docs apart from this PR.)

> . In the *Create content view* pop-up window, enter a name for the view in the *Name* field.

Author: ```suggestion
. In the *Create content view* window, enter a name for the view in the *Name* field.
```

{ProjectName} automatically completes the *Label* field from the name you enter.
. Optional: In the *Description* field, enter a description of the view.

> . Under the *Type* tab, select *Composite content view*.

Reviewer 2: ```suggestion
. On the *Type* tab, select *Composite content view*.
```

. Optional: If you want to automatically publish a new version of the Composite Content View when a Content View is republished, select the *Auto publish* checkbox.
. Click *Create content view*.

> . In the *Content views* tab, select the Content Views that you want to add to the Composite Content View, and then click *Add content views*.

Reviewer 2: ```suggestion
. On the *Content views* tab, select the Content Views that you want to add to the Composite Content View, and then click *Add content views*.
```

> . In the *Add content views* pop-up window, select the version of Content View.

Reviewer: ```suggestion
. In the *Add content views* pop-up window, select the version of each Content View.
```

> . In the *Add content views* pop-up window, select the version of each Content View.

Reviewer 2: ```suggestion
. In the *Add content views* window, select the version of each Content View.
```

. Optional: If you want to automatically update the Content View to the latest version, select the *Always update to latest version* checkbox.
. Click *Add*, then click *Publish new version*.
. Optional: In the *Description* field, enter a description of the Content View.

> . In the *Publish* pop-up window, set the *Promote* switch, then select the lifecycle environment.

Reviewer: ```suggestion
. In the *Publish* window, set the *Promote* switch, then select the lifecycle environment.
```
Since we decide to remove mentions to pop-up windows after all, please also remove this one. Once you've done that, this is good to go.

> . In the *Publish* pop-up window, rag the *Promote* slider, then select the lifecycle environment.

Reviewer: ```suggestion
. In the *Publish* pop-up window, drag the *Promote* slider, then select the lifecycle environment.
```

> . In the *Publish* pop-up window, drag the *Promote* slider, then select the lifecycle environment.

Reviewer 2: ```suggestion
. In the *Publish* pop-up window, set the *Promote* switch, then select the lifecycle environment.
```
Is "Promote" a slider (with more than two values) or a switch (true/false)?

Reviewer: It is a switch with true/false. Good catch!
